docs/Using-Fleet/fleetctl-CLI.md

```diff
@@ -196,7 +196,7 @@ Now that `fleetctl` and the Fleet server are configured, it can be helpful to cr
 
 ### Create an API-only user
 
-To create your new API-only user using `fleetctl user create` and passing a name, email and password and setting the `api-only` flag to `true`:
+To create your new API-only user using `fleetctl user create` and picking a name, email, and password and setting the `api-only` flag to `true`:
```

Comment:
@Desmi-Dizney good

Reply:
💯 

love that you're not being afraid of simplifying words that look like they could be developer lingo